python-backend/app/api/utils/models_mixins.py

```diff
@@ -56,7 +57,12 @@ def save(self, commit=True):
 
 
 class AuditMixin(object):
-    create_user = db.Column(db.String(60), nullable=False)
+    create_user = db.Column(db.String(60), nullable=False, default=User().get_user_username)
     create_timestamp = db.Column(db.DateTime, nullable=False, default=datetime.utcnow)
-    update_user = db.Column(db.String(60), nullable=False)
-    update_timestamp = db.Column(db.DateTime, nullable=False, default=datetime.utcnow)
+    update_user = db.Column(
+        db.String(60),
+        nullable=False,
+        default=User().get_user_username,
+        onupdate=User().get_user_username)
+    update_timestamp = db.Column(
+        db.DateTime, nullable=False, default=datetime.utcnow, onupdate=datetime.now)
```

Comment:
is there a reason one of these is now() and the other is utcnow()?

Reply:
good catch, i went through and changed all .now to ,utc now (except one nris comparison)